librz/io/meson.build

```diff
@@ -152,36 +152,8 @@ rz_io_dep = declare_dependency(link_with: rz_io,
                               include_directories: platform_inc)
 meson.override_dependency('rz_io', rz_io_dep)
 
-pkgconfig_mod.generate(rz_io,
-  subdirs: 'librz',
-  version: rizin_version,
-  name: 'rz_io',
-  filebase: 'rz_io',
-  requires: [
-    'rz_util',
-    'rz_socket'
-  ],
-  description: 'rizin foundation libraries',
-  variables: [
-    'plugindir=@0@'.format(rizin_plugins),
-    'datdir=@0@'.format(rizin_datdir_rz),
-  ],
-)
-
-if not is_static_libs_only
-  conf = configuration_data()
-  conf.set('RZ_VERSION', rizin_version)
-  conf.set('RIZIN_MODULE', rz_io.name())
-  conf.set('RIZIN_MODULE_DEPS', ' '.join(['rz_util', 'rz_socket', 'rz_hash', 'rz_crypto', 'rz_cons']))
-  conf.set('PACKAGE_RELATIVE_PATH', cmake_package_relative_path)
-  conf.set('INSTALL_INCDIR', rizin_incdir)
-  conf.set('INSTALL_LIBDIR', rizin_libdir)
-  conf.set('INSTALL_PLUGDIR', rizin_plugins)
-  conf.set('rizin_libname', rz_io.name())
-  cmake_mod.configure_package_config_file(
-    name: conf.get('rizin_libname'),
-    input: '../RzModulesConfig.cmake.in',
-    install_dir: rizin_cmakedir / conf.get('rizin_libname'),
-    configuration: conf,
-  )
-endif
+modules += { 'rz_io': {
+    'target': rz_io,
+    'dependencies': ['rz_util', 'rz_socket', 'rz_hash', 'rz_crypto', 'rz_cons'],
+    'plugins': [io_plugins]
```

Comment:
shall `plugins` be a list? or is it enough to do `'plugins': io_plugins`?

Reply:
`librz/bin/meson.build` currently needs it for `bin_xtr_...`.